config/dev-amo.js

```diff
@@ -25,5 +25,7 @@ module.exports = {
     },
   },
 
+  betaVersions: false,
```

Comment:
You should also add this to the local development config: https://github.com/wagnerand/addons-frontend/blob/e38bf9f021a196fb30b957cf64c8db5345dfaa92/config/development-amo.js